packages/transactions/src/decompile-transaction.ts

```diff
@@ -58,6 +59,54 @@ function getAccountMetas(message: CompiledMessage): IAccountMeta[] {
     return accountMetas;
 }
 
+export type LookupTables = { [lookupTableAddress: Address]: Address[] };
+
+function getAddressLookupMetas(
+    transactionLookups: ReturnType<typeof getCompiledAddressTableLookups>,
+    knownLookups: LookupTables,
+): IAccountLookupMeta[] {
+    // check that all message lookups are known
+    const transactionLookupAddresses = transactionLookups.map(l => l.lookupTableAddress);
```

Comment:
Nit: confusing naming here. What's a ‘transaction lookup?’ Next it's called a ‘message lookup.’

Maybe just call it what it is.

```ts
function getAddressLookupMetas(
    compiledAddressTableLookups: ReturnType<typeof getCompiledAddressTableLookups>,
    addressesByLookupTableAddress: AddressesByLookupTableAddress,
): IAccountLookupMeta[] {
```